src/Core/src/Platform/iOS/MauiPageControl.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using CoreGraphics;
+using UIKit;
+
+namespace Microsoft.Maui.Platform.iOS
+{
+	public class MauiPageControl : UIPageControl
+	{
+		const int DefaultIndicatorSize = 6;
+
+		IIndicatorView? _indicatorView;
+		bool _updatingPosition;
+
+		public MauiPageControl()
+		{
+			ValueChanged += MauiPageControlValueChanged;
+			if (NativeVersion.IsAtLeast(14))
```

Comment:
What is our min supported? Can we just assume 14?

Reply:
No min supported is 10